xml/System.Data.Common/DbConnection.xml

```diff
@@ -1527,12 +1527,12 @@
         <ReturnType>System.Data.StateChangeEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Occurs when the state of the event changes.</summary>
+        <summary>Triggered when the state of the connection changes.</summary>
```

Comment:
The original version is correct according to the style guide:
https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Event-related

 ```suggestion
        <summary>Occurs when the state of the connection changes.</summary>
```